main.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+#include <cmath>
+#include <stdio.h>
+
+using namespace std;
+
+int main()
+{
+    setlocale(0,"");
+    float left, right;
+    char znak;
+    cin >>left>>znak>>right;

```

Comment:
You have to inform user about actions he need to make. Now it's unclear what programm is expecting from him.